frontend/src/modules/app/ToolBar.tsx

```diff
@@ -114,11 +149,19 @@ export const ToolBar = ({
   const currentMode = useSelector(getParametersRequestMode);
   const popupEventLog = useSelector(getPopupEventLog, shallowEqual);
   const parameterRequestStandby = useSelector(getParametersRequestStandby, shallowEqual);
-  const alarmLimitsRequestStandby = useSelector(getAlarmLimitsRequestStandby, shallowEqual);
   const ventilating = useSelector(getParametersIsVentilating);
+  const alarmLimitsRequestStandby = useSelector(getAlarmLimitsRequestStandby);
+  const alarmLimitsCurrent = useSelector(getAlarmLimitsCurrent);
+  const alarmLimits = (alarmLimitsCurrent as unknown) as Record<string, Range>;
+  const alarmLimitsStandby = (alarmLimitsRequestStandby as unknown) as Record<string, Range>;
   const [isVentilatorOn, setIsVentilatorOn] = React.useState(ventilating);
   const [label, setLabel] = useState('Start Ventilation');
   const [isDisabled, setIsDisabled] = useState(false);
+  const [open, setOpen] = useState(false);
+  const setAlarmLimits = (data: Partial<AlarmLimitsRequest>) => {
```

Comment:
I'm a bit confused by the variable naming. Just to confirm, `commitStandbyRequest` should save data into `store.controller.alarmLimits.standby` (the AlarmLimitsRequestStandby), and not `store.controller.alarmLimits.request` (the AlarmLimitsRequest). If this is what you want, maybe it's better to rename `setAlarmLimits` to `setAlarmLimitsRequestStandby`.